dev/my/helmfile.yaml

```diff
@@ -0,0 +1,50 @@
+# single env example file
+# helm file dependencies
+# helm plugin install https://github.com/databus23/helm-diff
+# repositories to be installed
+repositories:
+  - name: datarepo-helm
+    url: https://broadinstitute.github.io/datarepo-helm
+
+# helm releases to be deployed
+releases:
+  - name: my-jade-create-secret-manager-secret    # release name
+    namespace: my   # target namespace
+    createNamespace: true
+    chart: datarepo-helm/create-secret-manager-secret   # chart name
+    version: 0.0.6   #    Chart version
+    missingFileHandler: Warn
+    values:
+      - create-secret-manager-secret.yaml      # Value files passed via --values
+  - name: my-jade-gcloud-sqlproxy    # release name
+    namespace: my   # target namespace
+    createNamespace: true
+    chart: datarepo-helm/gcloud-sqlproxy   # chart name
+    version: 0.19.7 #    Chart version
+    missingFileHandler: Warn
+    values:
+      - gcloud-sqlproxy.yaml    # Value files passed via --values
+  - name: my-jade-datarepo-api   # name of this release
+    namespace: my   # target namespace
+    createNamespace: true
+    chart: datarepo-helm/datarepo-api   # the chart name
+    version: 0.0.25    # chart version
+    missingFileHandler: Warn
+    values:
+      - datarepo-api.yaml   # Value files passed via --values
+  - name: my-jade-datarepo-ui   # name of this release
+    namespace: my   # target namespace
+    createNamespace: true
+    chart: datarepo-helm/datarepo-ui   # the chart name
+    version: 0.0.14    # chart version
+    missingFileHandler: Warn
+    values:
+      - datarepo-ui.yaml   # Value files passed via --values
```

Comment:
Totally your preference, but I recommend removing the "version: x.x.x" line from the API and UI charts. Since they're bumped on every commit to the repos, they're hard to keep up with. By removing the version, it will automatically use the latest. 
If not - here's the list of latest versions to manually update: https://github.com/broadinstitute/datarepo-helm/blob/master/charts/datarepo/Chart.lock